python/cudf/cudf/core/dataframe.py

```diff
@@ -5860,9 +5861,11 @@ def _reduce(
             )
             source = self._get_columns_by_label(numeric_cols)
             if source.empty:
-                return Series(index=self.index)
-
-        axis = source._get_axis_from_axis_arg(axis)
+                return Series(
+                    index=cudf.Index([], dtype="str")
```

Comment:
Curious why it should be `dtype="str"` and not `dtype=self.index.dtype`?

Reply:
Yes, it should be `self.columns.dtype`. I fixed it now 👍